.github/workflows/unit_tests.yml

```diff
@@ -7,8 +7,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ "3.9", "3.10" ]
-        os: [ ubuntu-latest, macos-12 ]
+        python-version: [ "3.9", "3.10", "3.11"]
+        os: [ ubuntu-latest, macOS-12 ]
```

Comment:
SUPER nitpicky of me, but casing here is fine in lowercase which is how GitHub references them too